toffy/mph_comp.py

```diff
@@ -152,7 +152,7 @@ def visualize_mph(mph_df, out_dir, regression: bool = False):
         io_utils.validate_paths(out_dir)
 
     # visualize the median pulse heights
-    plt.style.use('dark_background')
+    # plt.style.use('dark_background')
     # plt.title('FOV total counts vs median pulse height')
```

Comment:
These comments will need to be removed.

Reply:
@alex-l-kong Should be good now, just pushed the changes.